docs/sources/operators-guide/reference-http-api/index.md

```diff
@@ -1021,6 +1021,6 @@ Requires [authentication](#authentication).
 GET /compactor/delete_tenant_status
 ```
 
-Returns status of tenant deletion. Output format to be defined. Experimental.
+Returns status of tenant deletion.
```

Comment:
I just deleted "Output format to be defined." Not sure if we know the output format and then I could update these docs to include that.

Reply:
The response is a JSON like this:

```json
{
  "tenant_id": "<id>",
  "blocks_deleted": true
}
```

Where `blocks_deleted` is `true` if all the tenant's blocks have been deleted.

Reply:
added!